Tests/SwiftDocCTests/Rendering/DeprecationSummaryTests.swift

```diff
@@ -181,4 +181,66 @@ class DeprecationSummaryTests: XCTestCase {
       
       XCTAssertEqual(renderNode.deprecationSummary?.firstParagraph, expected)
   }
+    
+    func testDeprecationSummaryInDiscussionSection() throws {
+        let (bundle, context) = try testBundleAndContext(named: "BundleWithLonelyDeprecationDirective")
+        let node = try context.entity(
+            with: ResolvedTopicReference(
+                bundleIdentifier: bundle.identifier,
+                path: "/documentation/CoolFramework/CoolClass/coolFunc()",
+                sourceLanguage: .swift
+            )
+        )
+        
+        // Compile docs and verify contents
+        let symbol = node.semantic as! Symbol
+        var translator = RenderNodeTranslator(context: context, bundle: bundle, identifier: node.reference)
+        
+        guard let renderNode = translator.visit(symbol) as? RenderNode else {
+            XCTFail("Could not compile the node")
+            return
+        }
```

Comment:
nit: using XCTUnwrap is a more succinct way to assert that a value is non-nil
```suggestion
        let renderNode = try XCTUnwrap(translator.visit(symbol) as? RenderNode)
```